packages/wallet/src/services/types.ts

```diff
@@ -82,12 +82,12 @@ export enum StakeKeyStatus {
 export interface RewardAccount {
   address: Cardano.Address;
   keyStatus: StakeKeyStatus;
-  // Maybe add rewardsHistory$ and delegatee$ for each reward account too
+  delegatee: Delegatee;
```

Comment:
```suggestion
  delegatee?: Delegatee;
```
Semanticly reward accounts are not always delegated. This would be the case for a Stake Pool reward account, which is associated with the pool formally, but not delegating to it.

Reply:
The fields of `Delegatee` type are nullable:
```
export interface Delegatee {
  currentEpoch: Cardano.StakePool | null;
  nextEpoch: Cardano.StakePool | null;
  nextNextEpoch: Cardano.StakePool | null;
}
```
Should we make `nextNextEpoch` not nullable then?

Reply:
Stake pool reward accounts will likely never have a `Delegatee` though :thinking:

Reply:
I did make it optional and `nextNextEpoch` non-nullable as it will always be set if reward account is delegating